rawler/data/lenses.toml

```diff
@@ -225,18 +232,46 @@ model = "150-600mm f/5-6.3 DG OS HSM | S"
 focal_range = [[150, 1], [600, 1]]
 aperture_range = [[50, 10], [63, 10]]
 
-
 [[lenses]]
 key = "30mm F1.4 DC HSM | Art 013"
 make = "Sigma"
 model = "30mm f/1.4 DC HSM | A"
 focal_range = [[30, 1], [30, 1]]
 aperture_range = [[14, 10], [14, 10]]
 
-
 [[lenses]]
 key = "50mm F1.4 DG HSM | Art 014"
 make = "Sigma"
 model = "50mm f/1.4 DG HSM | A"
 focal_range = [[50, 1], [50, 1]]
 aperture_range = [[14, 10], [14, 10]]
+
+# Tamron lenses
+[[lenses]]
+key = "TAMRON SP 90mm F/2.8 Di VC USD MACRO1:1 F004"
+make = "Tamron"
+model = "SP 90mm F/2.8 Di VC USD Macro 1:1 F004"
+focal_range = [[90, 1], [90, 1]]
+aperture_range = [[28, 10], [28, 10]]
+
+[[lenses]]
+key = "TAMRON 10-24mm F/3.5-4.5 Di II VC HLD B023"
+make = "Tamron"
+model = "10-24mm f/3.5-4.5 Di II VC HLD (B023)"
+focal_range = [[10, 1], [24, 1]]
+aperture_range = [[35, 10], [45, 10]]
+
+[[lenses]]
+key = "TAMRON 17-35mm F/2.8-4 Di OSD A037"
+model = "17-35mm F/2.8-4 Di OSD A037"
+make = "Tamron"
+focal_range = [[17, 1], [35, 1]]
+aperture_range = [[28, 10], [40, 10]]
+
+# Venus Optics/Laowa lenses
+[[lenses]]
+key = "LAOWA 100mm F2.8 CA-Dreamer Macro 2x"
+model = "100mm F2.8 2X Ultra Macro APO"
```

Comment:
**Venus** is the manufacturer and **Laowa** the brand name, according to https://en.wikipedia.org/wiki/Venus_Optics
The lensfun database for lens correction contains only **Venus** as manufacturer and the model starts with **Laowa...*. (reference: https://wilson.bronger.org/lensfun_coverage.html)

To get get best compatibility, I would suggest to use the same naming scheme here, too.
So the model should be **Laowa 100mm F2.8 2X Ultra Macro APO** and the make just **Venus**.

Reply:
I've committed and pushed that change.

Danke schön!